opensearch/Dockerfile

```diff
@@ -1,7 +1,30 @@
 # Repo name: arynai/sycamore-opensearch
 
 ARG TAG=2.12.0
-FROM opensearchproject/opensearch:$TAG
+
+FROM gradle:jdk17 as build_ml_commons
+
+WORKDIR /build_ml
+RUN git clone https://github.com/opensearch-project/ml-commons.git
+WORKDIR /build_ml/ml-commons
+RUN git checkout -b 2.12 origin/2.12
+RUN sed -i 's#if (os.macOsX) {#if (true) {#' ml-algorithms/build.gradle
```

Comment:
I don't think we want this. If we're amd we want the _gpu one.
```
def arch = System.getProperty("os.arch")
if (arch == "aarch64") { ... }
```
might be the right thing.
https://github.com/apereo/cas/blob/bbfb3a9b4e95c17a45a72bd6e199343790755c17/build.gradle#L143